Testscripts/Linux/perf_lkp.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+###############################################################################
+#
+# Description:
+# Support distros: Ubuntu 18.04 
+#
+###############################################################################
+
+LKP_SCHEDULER_HACKBENCH_TESTS=("hackbench-100.yaml" "hackbench.yaml")
+LKP_SCHEDULER_PERF_BENCH_SCHED_PIPE_TESTS=("perf-bench-sched-pipe.yaml")
+LKP_SCHEDULER_SCHBENCH_TESTS=("schbench.yaml")
+LKP_DATABASE_OLTP_TESTS=("oltp.yaml")
+LKP_DATABASE_REDIS_TESTS=("redis.yaml")
+LKP_MEMORY_PERF_BENCH_NUMA_MEM_TESTS=("perf-bench-numa-mem.yaml")
+LKP_MEMORY_PMBENCH_TESTS=("pmbench.yaml")
+LKP_WORKLOAD_APACHEBENCH_TESTS=("apachebench.yaml")
+LKP_TESTS_ALL_ABOVE=("perf-bench-sched-pipe.yaml" "schbench.yaml" "oltp.yaml" "redis.yaml" \
+                     "perf-bench-numa-mem.yaml" "pmbench.yaml" "apachebench.yaml")
+LKP_OTHER_UNIXBENCH_TESTS=("unixbench.yaml")
+LKP_OTHER_AIM9_TESTS=("aim9.yaml")
+
+###############################################################################
+function install_dependencies() {
+    LogMsg "Installing dependency packages"
+
+    GetDistro
```

Comment:
Do you need another function call 'GetDistro'? I think it has done in the beginning of test script. Please check with @LiliDeng